File: src/exemplo06/brinquedo.py
class Brinquedo:
    _faixa_etaria = None
    _material = None

    def __init__(self, faixa_etaria, material):
        self._faixa_etaria = faixa_etaria
        self._material = material
    
    def getFaixaEtaria(self):
        return self._faixa_etaria
    
    def getMaterial(self):
        return self._material

    def __str__(self):
        return f'Faixa Etária: {self._faixa_etaria}, Material: {self._material}'

File: src/exemplo07/Brinquedo.java
package exemplo07;

public class Brinquedo {
    private int faixaEtaria;
    private String material;

    public Brinquedo(int faixaEtaria, String material) {
        this.faixaEtaria = faixaEtaria;
        this.material = material;
    }

    public int getFaixaEtaria() {
        return faixaEtaria;
    }

    public void setFaixaEtaria(int faixaEtaria) {
        this.faixaEtaria = faixaEtaria;
    }

    public String getMaterial() {
        return material;
    }

    public void setMaterial(String material) {
        this.material = material;
    }
}
